onedal/dal.cpp

```diff
@@ -102,6 +106,7 @@ namespace oneapi::dal::python {
 #else
     #ifdef ONEDAL_DATA_PARALLEL
     PYBIND11_MODULE(_onedal_py_dpc, m) {
+        init_sycl(m);
```

Comment:
Should not this be `sycl_interfaces` ?

Reply:
will change

Reply:
@samir-nasibli would I need to rename sycl.cpp to sycl_interfaces.cpp, and if so, do you want me to move the contents of sycl.cpp to sycl_interfaces.cpp?